Fix reclaim mannequin command without CSV (#795)
github/gh-gei: RELEASENOTES.md

- Added logic to check if a target repo exists before generating GHES archives

> - Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command

Reviewer: I suggest mentioning the command, enclosing the parameters in backticks and dropping the explanation of why it was broken (which only really matters for us!) :heart:

```suggestion
- Fixed reclaiming a single mannequin using `reclaim-mannequin` with the `--mannequin-user` and `--target-user` parameters
```

Author: @timrogers done

> - Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were were not being passed to the reclaim command

Reviewer 2: ```suggestion
- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command
```
